module/Company/src/Company/Form/EditJob.php

```diff
@@ -0,0 +1,185 @@
+            'attributes' => array(
+ //               'type'  => 'boolean',
+                'type' => 'Checkbox',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Location'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'website',
+            'attributes' => array(
+                'type'  => 'text',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Website'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'email',
+            'type' => 'Zend\Form\Element\Email',
+            'attributes' => array(
+                'type'  => 'text',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Email'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'phone',
+            'attributes' => array(
+                'type'  => 'text',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Phone'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'description',
+            'type' => 'Zend\Form\Element\Textarea',
+            'options' => array(
+                'label' => $translate->translate('Description'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'submit',
+            'attributes' => array(
+                'type'  => 'submit',
+                'value' => $translate->translate('Submit changes'),
+                'id' => 'submitbutton',
+            ),
+        ));
+        
+        $this->initFilters();
+    }
+    
+    protected function initFilters()
+    {
+        $filter = new InputFilter();
+
+
+        $filter->add(array(
+            'name' => 'name',
+            'required' => true,
+            'validators' => array(
+                array(
+                    'name' => 'string_length',
+                    'options' => array(
+                        'min' => 2,
+                        'max' => 127
+                    )
+                )
+            )
+        ));
+        
+        $filter->add(array(
+            'name' => 'website',
+            'required' => true,
+            'validators' => array(
+                array(
+                    'name' => 'regex',
+                    'options' => array(
+                        'pattern' => '_^(?:(?:https?|ftp)://)(?:\S+(?::\S*)?@)?(?:(?!10(?:\.\d{1,3}){3})(?!127(?:\.\d{1,3}){3})(?!169\.254(?:\.\d{1,3}){2})(?!192\.168(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\x{00a1}-\x{ffff}0-9]+-?)*[a-z\x{00a1}-\x{ffff}0-9]+)(?:\.(?:[a-z\x{00a1}-\x{ffff}0-9]+-?)*[a-z\x{00a1}-\x{ffff}0-9]+)*(?:\.(?:[a-z\x{00a1}-\x{ffff}]{2,})))(?::\d{2,5})?(?:/[^\s]*)?$_iu'
+                     )     
```

Comment:
Lijkt mij wel heel zeker ja. Of, comment de hele regex. Of eigenlijk, toch maar de Uri validator. Regexes van deze grootte zijn een beetje heel erg extreem kut.